studybuddies/src/app/components/login/login.component.ts

```diff
@@ -13,24 +17,50 @@ import {HttpClient} from '@angular/common/http'
 })
 export class LoginComponent implements OnInit {
 
-  user: User = new User();
-  loginError = null;
-  logged: boolean;
+  constructor(
+    private authService: AuthService,
+    private router: Router,
+    private tokenStorage: TokenStorageService
+  ) 
+  {}
 
-  constructor(private loginService: LoginService, public router: Router, private cookieService: CookieService, public translate:TranslateService) { }
+  username = '';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
  username = "";
```